### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This PR includes the following updates:

- Custom schema integration
- Github pages docs addition
- Postgres compatibility
- Circle, yml, and readme minor updates
- Version upgrade to v0.2.0

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -1,16 +1,19 @@
 name: 'pardot_source'
 version: '0.2.0'
```
Comment (fivetran-jamie) [inline dbt_project.yml:2]: should we update to 0.3.0 bc of the custom schemas?

Comment (fivetran-joemarkiewicz) [reply dbt_project.yml:2]: The version was incorrectly set originally. The tag on the release is actually v0.1.0 currently so keeping v0.2.0 takes into account the breaking change from custom schemas.

Comment (fivetran-jamie) [reply dbt_project.yml:2]: ah sweet

**Review by fivetran-jamie — APPROVED** [review]
looks good to me!

**Pull request merged**